daemon/state_test.go

```
@@ -187,27 +177,15 @@ func (ds *DaemonSuite) TestSyncLabels(c *C) {
 
 	// Let's make sure we delete all labels from the kv store first
 	ep2 := epsWanted[1]
-	ep2id := ep2.StringID()
-	ep2SecLabelID := ep2.SecLabel.ID
-	hash := ep2.SecLabel.Labels.SHA256Sum()
-	ds.d.DeleteIdentityBySHA256(hash, ep2id)
+	ep2.SecLabel.Release()
```

Why is the error here ignored? I prefer explicitly ignoring errors with `_ = ep2.SecLabel.Release()` for example, as it makes it clear that the caller intentionally is ignoring the returned value.